src/common/gui/base/Dialog.ts

```diff
@@ -716,6 +738,28 @@ export class Dialog implements ModalComponent {
 		})
 	}
 
+	static reminderDialog(title: string, message: string | lazy<Children>, image: string, buttonAttrs: Array<ButtonAttrs>): Dialog {
+		return new Dialog(DialogType.Reminder, {
+			view: () => [
+				m(".dialog-contentButtonsBottom.text-break.scroll", [
+					m(".h2.pb", title),
+					m(".flex-direction-change.items-center", [
+						m("#dialog-message.pb.selectable", typeof message == "function" ? message() : message),
+						m("img[src=" + image + "].dialog-img.mb.bg-white.border-radius", {
```

Comment:
we should not use attributes, especially computed ones, in the string part of hyperscript. It's often insecure and error prone. I think it's in our guidelines somewhere but I'm unsure.

I know it was there before but now that it's an actual parameter it's more acute and also a good reason to change it.